content/tutorials/manage-the-rippled-server/installation/system-requirements.md

```diff
@@ -25,12 +25,12 @@ Amazon EC2's `m3.large` VM size may be appropriate depending on your workload. A
 
 ## Recommended Specifications
 
-For best performance in enterprise production environments, Ripple recommends running `rippled` on bare metal with the following characteristics:
+For best performance in enterprise production environments, it is recommended to run `rippled` on bare metal with the following characteristics:
 
 - Operating System: Ubuntu (LTS) or CentOS or RedHat Enterprise Linux (latest release)
-- CPU: Intel Xeon 3+ GHz processor with 4 cores and hyperthreading enabled
-- Disk: SSD (10,000 IOPS or better)
-- RAM: 32 GB
+- CPU: Intel Xeon 3+ GHz processor with 8 cores and hyperthreading enabled
+- Disk: SSD / NVMe(10,000 IOPS or better)
```

Comment:
At Ripple we've been thinking of bumping the IOPS number as well since 10,000 is pretty low for modern hardware. Can follow up with that.